src/builder_state.rs

```diff
@@ -971,12 +995,8 @@ impl<TYPES: NodeType> BuilderState<TYPES> {
             da_proposal_receiver: self.da_proposal_receiver.clone(),
             qc_receiver: self.qc_receiver.clone(),
             req_receiver,
-            da_proposal_payload_commit_to_da_proposal: self
-                .da_proposal_payload_commit_to_da_proposal
-                .clone(),
-            quorum_proposal_payload_commit_to_quorum_proposal: self
-                .quorum_proposal_payload_commit_to_quorum_proposal
-                .clone(),
+            da_proposal_payload_commit_to_da_proposal: HashMap::new(),
+            quorum_proposal_payload_commit_to_quorum_proposal: HashMap::new(),
```

Comment:
Why this change?

Reply:
Potential memory leak if we're seeing multiple competing proposals (e.g. there's a malicious leader, so we shouldn't ever hit that, but still) that compounds as the outstanding number of possible parent proposals.